### ISSUE TITLE
upgrade Music_Calculator

### ISSUE THREAD

**Issue by EdmundCampion** [CNMAT/CNMAT-MMJ-Depot#59]
The Music Calculator patch in Depot is too large in view and missing a lot of needed functions that never got implemented.  BPM and Frequency section is fine.  

Please reduce size of "file size" considerable, but don't lose the ability to post larger numbers in the various number boxes.  

Add super-divisions to the "ms per" field.  Add half-note, dotted half-note, and whole note

Add the following features;
A clock-time addition/subtraction that looks something like this:
Hour:	Minute:	Second:	
		: 	   Time
		: 	   Times to Add or Subtract 
		: 	   Number of Times to Multiply or Divide

: 	   Final Time in Hour Minute Second

**Comment by equilet** [comment]
I think this patch is something that should be rewritten in odot. Let me know if that also makes sense to you.

**Comment by EdmundCampion** [comment]
no, don't bother.  We can look over things side by side when we review all Depot.
